### PR TITLE
Update 404 error illustrations

### PR DESCRIPTION
**Pull request by kjellr**
Updating our 404 error illustrations to match the new WordPress.com illustration style.

Half of issue #13038

---

Current 404 Illustration: 
404-drake

Updated 404 Illustration:

### CODE REVIEW

**Comment by kjellr** [comment]
## Screenshots: 

General 404 page:
screen shot 2017-04-24 at 2 42 12 pm

Reader / Manage (Empty)
screen shot 2017-04-24 at 2 31 30 pm

Reader / Feed not found
screen shot 2017-04-24 at 2 38 22 pm

---

Still working on a few more places we use the old 404 image.

**Comment by kjellr** [comment]
Plugins / Access control
screen shot 2017-04-25 at 8 13 18 am

**Comment by kjellr** [comment]
I'll break the 500 errors out into their own PR to keep this more clean.

**Comment by kjellr** [comment]
In dd2e50d, I darkened the color of the browser window to match the colors from #12374. This works better against our light gray backgrounds.

Updated screenshots:

General 404 page
screen shot 2017-04-25 at 10 11 09 am


Reader / Manage (Empty)
screen shot 2017-04-25 at 10 07 04 am

Reader / Feed not found
screen shot 2017-04-25 at 10 10 11 am

Plugins / Access control
screen shot 2017-04-25 at 10 06 49 am

**Comment by kjellr** [comment]
In fc6777b, I updated the image in a couple files that I can't find in the front-end:

- `client/login/magic-login/emailed-login-link-expired.jsx`: Your magic link has expired
- `client/my-sites/plugins/plugin.jsx`: Plugin cannot be found

Would someone mind checking these and/or telling me how to verify them in the front-end? The image swap is likely no problem at all, but a couple previous updates needed minor CSS tweaks, so I want to be sure. 

cc @jblz + @jhnstn in case you know — it looks like you two have recent commits in those files. 

Thanks!

**Comment by jblz** [comment]
@kjellr, the magic login feature isn't live yet and needs a bit of design work anyhow, so, you're ok to make the change there. Thanks for the ping.

**Comment by kjellr** [comment]
I figured out a way to test the "plugins not found" error screen, so we should be set. 

@foletto — Thank you for reviewing the [500 Error Illustration PR](https://github.com/Automattic/wp-calypso/pull/13413) last week. Would you mind taking a look at this one too?

Thanks!

## Screenshots + Testing instructions:

**General 404 page:**

- Visit http://calypso.localhost:3000/404

screen shot 2017-04-24 at 2 42 12 pm

---

**Reader Manage / Empty:**

- Start a test account
- Visit http://calypso.localhost:3000/following/edit
- Delete any auto-followed sites. When your list of followed sites is empty, you should see the illustration below. 

screen shot 2017-04-24 at 2 31 30 pm

---

**Reader / Feed not found**

- Visit http://calypso.localhost:3000/read/feeds/testing (`testing` in the URL can be any term)

screen shot 2017-04-24 at 2 38 22 pm

---

**Plugins / Access Control**

- While logged in as a non-administrator for a Jetpack site, try visiting http://calypso.localhost:3000/plugins/{site-url} (where `{site-url}` is the url of the Jetpack site)

screen shot 2017-04-25 at 8 13 18 am

---

**Plugins / Plugin cannot be found**

- In `client/my-sites/plugins/plugin.jsx`, temporarily replace lines 392-416 with `return this.getPluginDoesNotExistView( selectedSite );` to force the 404 view.
- Visit any plugin page (for example, http://calypso.localhost:3000/plugins/akismet/{site-url}, where `{site-url}` is the url of the Jetpack site)

screen shot 2017-05-03 at 8 52 19 am

**Comment by folletto** [comment]
Design looks good 👍

File: public/images/illustrations/illustration-404.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...="st14" d="M254.7 120.1l.1.8"/><path class="st13" d="M254.3 125.4c.4.1.8 0 1.2-.1l.6-.3c.2-.1.3-.3.4-.4.1-.2.3-.2.5-.1.1.1.2.2.2.3 0 .4-.2.7-.4 1-.2.3-.5.4-.8.6-.3.1-.7.2-1 .1-.3 0-.7-.1-1-.4-.1-.1-.1-.3 0-.5 0-.2.1-.2.3-.2zm10.9-13.1c.4 3.3 7.8 4.7 7.8-1.4s-8.7-6.2-7.8 1.4z"/><path class="st2" d="M265.7 112.3c.1.9.9 1.6 1.8 1.9s1.9.3 2.7 0c.8-.3 1.3-1 1.6-1.8.2-.8.3-1.9.1-2.7-.2-.9-.7-1.6-1.4-2.1s-1.7-.6-2.5-.3-1.6 1-1.9 1.9c-.5 1-.5 2-.4 3.1zm-1 .1c-.1-1.1-.1-2.4.3-3.5s1.2-2.3 2.4-2.8 2.7-.5 3.9.1c1.2.6 2.1 1.8 2.5 3.1.2.6.2 1.3.2 1.9s-.1 1.3-.3 1.9c-.2.6-.6 1.3-1.1 1.7-.5.5-1.2.8-1.8 1-1.3.3-2.6.1-3.7-.5-.6-.3-1.1-.6-1.5-1.1-.5-.5-.9-1.2-.9-1.8z"/><path class="st15" d="M289.8 85.7l-4.9 12.6"/><path class="st2" d="M290.3 85.9c-.2 1.1-.5 2.2-.8 3.3-.4 1.1-.7 2.1-1.2 3.2l-1.3 3.1c-.5 1-1 2-1.6 3-.1.2-.5.3-.7.2-.2-.1-.3-.3-.2-.5.2-1.1.6-2.2.9-3.3.3-1.1.7-2.1 1.1-3.2s.8-2.1 1.3-3.1 1-2 1.6-3c.2-.2.5-.3.7-.1.2 0 .2.2.2.4z"/><path class="st13" d="M254.5 111.2c-3.2 0-1.8 1.5-1.8 1.5s-.9.3 0 1.5 6.6 5.5 10.8 6.6c4.3 1 3.8 1.5 4 1 .3-.5 1.3-3.8-1.2-7.4s-4.7-5.4-8-4.8l-3.8 1.6z"/><path class="st7" d="M310 205.6c-5.4.2-21.7 1.2-24.2 2.1-2.4.9-2.6 1.2.3 1.9 3 .7 23.8 0 23.8 0v-4h.1z"/></svg>
```
Comment (folletto) [inline public/images/illustrations/illustration-404.svg]: It contains "drake-404" as title, we should remove titles, as it's already embedded in the page, or at least give a more generic one. :)

**Comment by folletto** [comment]
Code looks good too, with the exception of the `title` note :)

File: public/images/illustrations/illustration-404.svg (hunk 1 above)
Comment (kjellr) [inline public/images/illustrations/illustration-404.svg]: Thanks! I've changed it to "Not Found" instead.

**Comment by folletto** [comment]
Ok, code 👍  too.

Rebase and merge 🚢

**Pull request merged**